.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  - oraclejdk8
```

Comment:
Would it be useful to have it email the team if the build is broken?  I believe by default it just emails the committer and the commit author.